File: python/01-sequencias/main015.py
''' Crie um programa que leia o número de dias trabalhados em um mês e mostre o 
salário de um funcionário, sabendo que ele trabalha 8 horas por dia e ganha R$25 
por hora trabalhada.

Data 29/09/2025 -> 8ª Dia
'''

dia = int(input('Digite o número de dias trabalhados em um mês: '))

hora = dia * 8
salrio = hora * 25

print(f'No mês o funcionário trabalhou {hora} e ele ganha {salrio}')

File: c/01-sequencias/main015.c
/*Crie um programa que leia o número de dias trabalhados em um mês e mostre o 
salário de um funcionário, sabendo que ele trabalha 8 horas por dia e ganha R$25 
por hora trabalhada.

Data 29/09/2025 -> 8ª Dia
*/

#include <stdio.h>

int main() {

    int dia;
    int hora;
    float salario;

    scanf("%d",&dia);

    hora = dia * 8;
    salario = hora * 25;

    printf("No mes o funcionario trabalhou %d e ele ganha %f",hora,salario);
    return 0;
}